constrained_ik/src/moveit_interface/constrained_ik_planner_plugin.cpp

```diff
@@ -115,12 +125,23 @@ namespace constrained_ik
 
   void CLIKPlannerManager::managerDynamicReconfigureCallback(CLIKPlannerDynamicConfig &config, uint32_t level)
   {
-    typedef std::map<std::pair<std::string, std::string>, constrained_ik::CLIKPlanningContextPtr>::const_iterator it_type;
+    typedef std::map<std::pair<std::string, std::string>, planning_interface::PlanningContextPtr>::const_iterator it_type;
 
     config_ = config;
     for (it_type it = planners_.begin(); it != planners_.end(); it++)
     {
-      it->second->setPlannerConfiguration(config_);
+      if (it->second->getName() == JOINT_INTERP_PLANNER)
+      {
+        boost::shared_ptr<JointInterpolationPlanner> planner = boost::static_pointer_cast<JointInterpolationPlanner>(it->second);
+        planner->setPlannerConfiguration(config_.joint_discretization_step, config_.debug_mode);
+      }
+
+      if (it->second->getName() == CARTESIAN_PLANNER)
+      {
+        boost::shared_ptr<CartesianPlanner> planner = boost::static_pointer_cast<CartesianPlanner>(it->second);
+        planner->setPlannerConfiguration(config_.translational_discretization_step, config_.orientational_discretization_step, config_.debug_mode);
+      }
+
```

Comment:
Would it make sense to split the dynamic reconfigure parameters  into two separate ones (joint interpolator and clik) that way it would avoid confusion when tweaking a parameter that's meant to affect a planner that isn't the currently selected one.